### DISCUSSION TITLE
Beatriz Milz - Introducción a GitHub Actions para usuarias de R

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#41] [General]
# Beatriz Milz - Introducción a GitHub Actions para usuarias de R

https://beamilz.com/posts/series-gha/2022-series-gha-1-what-is/es/index.html?q=gith

**Comment by Alex-Serafim** [comment]
Parabéns pelo contéudo, teria a possibilidade de trabalhar com meu blog por meio de dois computadores?
Eu queria utilizar um no trabalho e outro em casa.
Não estou conseguindo saber o meio de fazer isso, Só sei fazer em um. Obrigado
